generate.go

```diff
@@ -9,9 +9,8 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/Sirupsen/logrus"
+	"github.com/opencontainers/ocitools/libgen"
```

Comment:
On Mon, Jul 11, 2016 at 11:36:11AM -0700, hmeng-19 wrote:
> @wking , in your proposal, there are two `generate.go` - one is
> under `cmd/ocitools`, another is under `generate`. What is their
> difference? The same problem happens to `validate.go`.

The cmd/ocitools stuff is just thin bindings between the library code
(under generate/, validate/, etc.) and urfave/cli.

> I think all the files under `generate` and `validate` should be
> libraries, right?

Yup.